tests/python/unittest/test_numpy_op.py

```diff
@@ -3021,6 +3021,7 @@ def hybrid_forward(self, F, a, b, *args, **kwargs):
                       [[_np.float16, _np.float32, _np.float64], [_np.int32]]),
         'power': (1.0, 3.0, [lambda y, x1, x2: _np.power(x1, x2 - 1.0) * x2],
                              [lambda y, x1, x2: _np.power(x1, x2) * _np.log(x1)]),
+        'gcd': (-100, 100, [None], None, [[_np.int32]]),
```

Comment:
Can you tell me what is the test case here ? I didn't understand i/p and expected o/p